### DISCUSSION TITLE
Minimize homebrew install time for Docker?

### DISCUSSION THREAD

**Discussion by richb-hanover** [Homebrew/discussions#3688] [Everyday usage]
I have inherited a workflow that uses `brew` to install other tools. I am working to "Dockerize" it to simplify its evolution.

The existing process installs brew using the shell script from the home page. That's good. But the process has taken over 25 minutes so far... (I realize that this is a one-time step if I'm doing it right - the result is stashed in a layer that won't have to be re-computed.) 

My question: Is there a way to minimize the time required for `Tapping homebrew/core`? (Especially in a Docker context/container?) Many thanks.

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Installation#skip-tap-cloning-beta

**Comment by richb-hanover** [comment]
Thanks for the speedy response. I'm seeing equivocal results - I'll send data once I convince myself that it's meaningful...
